doc/tutorial.txt

```diff
@@ -99,7 +99,7 @@ is False.
 * lock_pages : Boolean. Lock the mem page in RAM. Locking the page in RAM
 ensures that your RT thread will not be stalled until a page is moved from swap
 to RAM.  The lock of the page is only possible for non CFS tasks. Default value
-is False.
+is True.
```

Comment:
Forced False for CFS, though. I think we should say that.

Reply:
so "Default value is True except for CFS it's FALSE? Does it make sense to change the default to false rather than fixup the documentation?

Reply:
I was about to fix this up when I noticed that it does state that "The lock of the page is only possible for non CFS Tasks." I guess is clear enough?